ivy/functional/backends/jax/experimental/layers.py

```diff
@@ -597,9 +597,7 @@ def interpolate(
     mode = (
         "nearest"
         if mode == "nearest-exact"
-        else "bicubic" "bicubic"
-        if mode == "bicubic_tensorflow"
-        else mode
+        else "bicubicbicubic" if mode == "bicubic_tensorflow" else mode
```

Comment:
This is one of the simplification of string preprocessing of black, I guess it was a bug in our initial code.